### PR TITLE
Fix transitions

### PR DESCRIPTION
**Pull request by andreasimonetti**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -60,9 +60,7 @@ export default class Machine extends erx.Bus<State> {
       this.transitioning = true;
       p(tFn(this, data)).then((change) => {
         this.transitioning = false;
-        if (change) {
-          finish(next);
-        }
+        change && finish(next);
```
Comment (evenius) [inline src/index.js:63]: Isn't this still gonna get weird if change is promise?

Comment (andreasimonetti) [reply src/index.js:63]: nop, p(tFn()) will always return a promise so change will be a value .then ...

**Comment by francisroshan** [comment]
Don't make promises you cannot keep @andreasimonetti :p

P.S. Please remove me from this account!

**Comment by bartoszadamczyk** [comment]
Hello @francisroshan.
This repo is public and you are following it, feel free to hit unwatch button.

**Comment by andywilliams** [comment]
Hi Francis!

On Mon, Oct 23, 2017 at 12:19 PM, Bartosz Adamczyk <notifications@github.com

**Comment by francisroshan** [comment]
Hey @andywilliams. How are you?

@lenddi Oops did not know it was public. Highlighted since I thought it may be a security issue. Thanks!

**Comment by bartoszadamczyk** [comment]
Not a problem at all! :) that would be a great spot if repo would be private :D

**Pull request merged**